guides/automations.rst

```diff
@@ -750,6 +776,33 @@ Configuration options:
 - **interval** (**Required**, :ref:`config-time`): The interval to execute the action with.
 - **then** (**Required**, :ref:`config-action`): The action to perform.
 
+
+Timers and timeouts
+-------------------
+
+While ESPHome does not provide a construction for timers, you can easily implement them by
+combining ``script`` and ``delay``. You can have an absolute timeout or sliding timeout by
+using script modes ``single`` and ``restart`` respectively.
+
+.. code-block:: yaml
+
+    script:
+      id: hallway_light_script
```

Comment:
Would be good to add a `- ` here to make it a list. Most people probably want to use multiple scripts this keeps the docs consistent (even though both are accepted by the validator)

```suggestion
    - id: hallway_light_script
```